### PR TITLE
Conservative Q-Learning

### PR DESCRIPTION
**Pull request by AntoineTheb**

### CODE REVIEW

File: article/_posts/2021-06-09-CQL.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+layout: review
+title: "Conservative Q-Learning for Offline Reinforcement Learning"
+tags: reinforcement
+author: ""
```
Comment (jhlegarreta) [inline article/_posts/2021-06-09-CQL.md:5]: Should be your name here.

File: article/_posts/2021-06-09-CQL.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+cite:
+    authors: "Aviral Kumar, Aurick Zhou, George Tucker, Sergey Levine"
+    title:   "Conservative Q-Learning for Offline Reinforcement Learning"
+    venue:   "arXiv"
+pdf: "https://arxiv.org/abs/2006.04779"
```
Comment (jhlegarreta) [inline article/_posts/2021-06-09-CQL.md:10]: I tend to put the link to the PDF as the tag says, but not sure whether that is general rule or a preference.

**Review by jhlegarreta — APPROVED** [review]
As always, very nice stuff.

Have not checked the equations, but looks good.

A few inline comments.

File: article/_posts/2021-06-09-CQL.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+They call their MaxEnt version $$CQL(\mathcal{H})$$.
+
+## Data
+
+The authors test their results on the OpenAI Gym suite of robotic control environments as well the Arcade Learning Environment.
+
+# Results
+
+![](/article/images/CQL/res1.jpeg)
```
Comment (lemairecarl) [inline article/_posts/2021-06-09-CQL.md:61]: wondering what's the logic of the bold numbers here. Top result and those that are close?

File: article/_posts/2021-06-09-CQL.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+![](/article/images/CQL/res2.jpeg)
+![](/article/images/CQL/res3.jpeg)
+
+# Conclusions
+
+CQL provides a simple modification to the standard Q-Learning or Actor-Critic updates which greatly improve offline reinforcement learning performances.
+
+# Remarks
+
+- The paper is still in its first preprint version on arXiv and so contains several typos, formatting errors and nebulous passages. However, the world of RL moves so fast that its results are already used in several other papers and that CQL is already described as state-of-the-art for Offline RL.
```
Comment (lemairecarl) [inline article/_posts/2021-06-09-CQL.md:71]: damn son

**Review by lemairecarl — APPROVED** [review]

File: article/_posts/2021-06-09-CQL.md (hunk 3 above)
Comment (AntoineTheb) [reply article/_posts/2021-06-09-CQL.md:61]: Yes

File: article/_posts/2021-06-09-CQL.md (hunk 2 above)
Comment (AntoineTheb) [reply article/_posts/2021-06-09-CQL.md]: Fixed in 43aa5a7

File: article/_posts/2021-06-09-CQL.md (hunk 1 above)
Comment (AntoineTheb) [reply article/_posts/2021-06-09-CQL.md]: Fixed in 43aa5a7

**Pull request merged**